docs/how-to/configure.md

```diff
@@ -70,7 +70,37 @@ This enables you to:
 - Reduce storage requirements by adjusting resolution for less critical components
 - Align your monitoring setup with the specific needs of your environment
 
-For information on enabling this feature via API, see the [**Change Postgres Exporter endpoint**](https://percona-pmm.readme.io/reference/changepostgresexporter) in the API documentation.
+As an example, suppose you want to change the medium resolution to 15s for a PostgreSQL server.
+
```

Comment:
1. Identify the appropriate API endpoint:

     - for PostgreSQL: `ChangePostgresExporter`
     - for MySQL: `ChangeMySQLdExporter`
     - for MongoDB: `ChangeMongoDBExporter`

2. Locate the `agent_id` of the exporter you want to modify. You can find this in the [Inventory dashboard](../details/dashboards/dashboard-inventory.md) under the **Monitoring** column for the target service.
3. Set the desired resolution using `hr` (high), `mr` (medium), or `lr` (low) in the `metrics_resolutions` field.